plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -269,12 +275,15 @@ function setupInternalRoutes(
       params: { instanceId },
     } = req;
 
-    await routerApi.v1
-      .abortWorkflow(instanceId)
-      .then(() => res.status(200).send())
-      .catch(error => {
-        res.status(500).send(error.message || INTERNAL_SERVER_ERROR_MESSAGE);
-      });
+    try {
+      await routerApi.v1.abortWorkflow(instanceId);
+      res.status(200).send();
+    } catch (error) {
+      res
+        .status(500)
+        .contentType('plain/text')
```

Comment:
why plain/text on the abort response?

Reply:
Because, when an error occurs (Replying with status 500), we are delivering to the client a plain text message, not an object ([Line 285](https://github.com/janus-idp/backstage-plugins/pull/1681/files#diff-afc3e7a4a5383fae292402adaf30b3ac1af05826dee9a024a693cd95d247ff15R285)).

Reply:
> Because, when an error occurs (Replying with status 500), we are delivering to the client a plain text message, not an object ([Line 285](https://github.com/janus-idp/backstage-plugins/pull/1681/files#diff-afc3e7a4a5383fae292402adaf30b3ac1af05826dee9a024a693cd95d247ff15R285)).

don't need a body at all I guess. but +1